CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Change Log
 
 
+## 1.0.0 - 2016-01-26
+
+### Removed
+
+- Stability config from composer
```

Comment:
i think we should actually not clutter the changelog with this. users of the library do not see any effect of that change, its only used when installing the lib for development.